tests/pytest_framework/src/syslog_ng/syslog_ng_cli.py

```diff
@@ -25,7 +25,7 @@
 from src.syslog_ng.syslog_ng_executor import SyslogNgExecutor
 from src.syslog_ng.console_log_reader import ConsoleLogReader
 from src.syslog_ng_ctl.syslog_ng_ctl import SyslogNgCtl
-from src.common.blocking import wait_until_true
+from src.common.blocking import wait_until_true, wait_until_false
```

Comment:
I think this belongs to the previous commit (`pytest_framework: look for process instead of stats when exiting`).